docs/downstream/odh-install-crc.md

```diff
@@ -0,0 +1,83 @@
+# Installing ODH using ArgoCD
+
+## Preparation
+
+First make sure that you have the OpenDataHub operator available in your OpenShift.
+
+```bash
+oc get packagemanifests -n openshift-marketplace | grep opendatahub-operator
+```
+
+If it's not present, then you need to troubleshoot the marketplace: https://github.com/operator-framework/operator-marketplace/issues/344
+
+
+Then we need to create a projects named `odh-operator` and `odh-deployment`. (The name of the project is hard-coded in the `kustomize` files in https://github.com/anishasthana/my-odh-install and you can change is there using `kustomize`.)
+
+```bash
+oc new-project odh-operator
+oc new-project odh-deployment
+```
+
+Our ArgoCD SA `argocd-manager` needs to have access to the new project. On CRC I suggest giving `argocd-manager` access to all projects by creating a ClusterRoleBinding to a ClusterRole.
+
+`oc apply -f examples/argocd-cluster-binding.yaml`
+
+Then you need to change the `dev-cluster` to include all projects/namespace. (This is done by actually removing the value of `namespace`).
+
+```bash
+oc patch secret dev-cluster-spec -n aicoe-argocd-dev --type='json' -p="[{'op': 'add', 'path': '/data/namespace', 'value':''}]"
+```
+
+Be aware that if you change this value in the ArgoCD UI, you might loose the stored credentials for the cluster due to a bug in ArgoCD.
+
+## Creating the ArgoCD applications
+
+Now we can proceed with creating the "applications" in ArgoCD. We will create 2 applications:
+
+  1. For the ODH operator. (Installs the ODH operator itself `odh-operator`.)
+  1. For the ODH deployment. (Installs ODH components into `odh-deployment`.)
+
+### Creating the ODH opetaror
+
+Go to https://argocd-server-aicoe-argocd-dev.apps-crc.testing/applications and click "New App". Enter these values:
+
+| Field            | Value                                                |
+|------------------|------------------------------------------------------|
+| Project:         | Default                                              |
+| Cluster:         | dev-cluster (https://api.crc.testing.6443)           |
+| Namespace:       | odh-operator                                         |
+| Repo URL:        | https://github.com/anishasthana/my-odh-install.git   |
+| Target revision: | HEAD                                                 |
+| Path:            | odh-operator/base                                    |
```

Comment:
I think it would be better to provide the `Application` yaml snippet here, instead of a table of what user should click and type in the UI. What do you think?

Swap it for something like:

```
cat <<EOF | oc apply -f -
apiVersion: argoproj.io/v1alpha1
kind: Application
metadata:
  name: odh
spec:
  destination:
    namespace: odh-operator
    server: https://api.crc.testing.6443
  project: default
  source:
    path: odh-operator/base
    repoURL: https://github.com/anishasthana/my-odh-install.git
    targetRevision: HEAD
  syncPolicy:
    automated:
      prune: true
      selfHeal: true
    syncOptions:
      - Validate=false
EOF
```

Reply:
yes, it makes sense

Reply:
But I'd rather put the resources into the repo than have them in-line in the doc. It will be easier to (automatically) test if things keep working. WDYT?

Reply:
+1 on the yaml snippet.

Put the yaml into the repo here in `/docs/downstream` for now. 
In the future I think the ODH specific steps should be in a separate repo anyways